src/components/icon/Icon.tsx

```diff
@@ -57,6 +58,12 @@ const Icon = styled.i
     }
     
     ${getIconSizes};
+    ${(props: ThemedStyledProps<PIcon, TTheme>): FlattenSimpleInterpolation | null =>
+        props.color
+            ? css`
+                  color: ${props.theme!.palette[props.color!].main};
```

Comment:
Don't we need a fallback here?

Reply:
the fallback is the `inherit` that is the default behavior (it is 'hard-coded' in the styles section)